apps/src/templates/codeDocs/EmbeddedBlock.jsx

```diff
@@ -23,13 +24,13 @@ export default function EmbeddedBlock({blockName, link}) {
 
   return (
     <div>
-      <a href={link}>
+      <Link href={link}>
```

Comment:
i'm going to try to sneak in some small accessibility wins with this 😄 right now, screenreaders will read this element as "[not] visited, link, [whatever-href-is]", but if you add an "aria-label" to the child (since the child isn't a text node), the screenreader output will be more digestible: "[not] visited, link, [aria-label]"

an example is probably easier:
```javascript
// screenreader: "visited, link, Go to Code Studio"
// anything that can resolve to a text node doesn't need an aria-label
<Link href="https://studio.code.org">Go to Code Studio</Link>

// screenreader: "visited, link, https://studio.code.org"
// what we have here
<Link href="https://studio.code.org">
  <div />
</Link>

// screenreader: "visited, link, Go to Code Studio"
<Link href="https://studio.code.org">
  <div aria-label="Go to Code Studio" />
</Link>
```

in this case, maybe the label should be `blockName`?

we may want to enforce this in `<Link>` at some point, but it's kind of a gray area because aria-label isn't always required.

Reply:
Yes good thought! thanks!

Reply:
I went a bit farther and added a prop to pass in the name of the expression, which is a bit more human readable